api/src/main/resources/workbench-api.yaml

```diff
@@ -1527,6 +1527,22 @@ paths:
           description: Internal Error
           schema:
             "$ref": "#/definitions/ErrorResponse"
+
+  "/v1/cron/updateResearchPurposeReview":
+    get:
+      tags:
+      - offlineWorkspace
+      - cron
+      description: Updates each Workspace Research Purpose Review
```

Comment:
This makes it sound like it's reviewing the research purpose, which is actually handled by `reviewWorkspace`.